docker/development.yml

```diff
@@ -90,6 +90,10 @@ services:
       context: ../src/api/parser
       cache_from:
         - docker.cdot.systems/parser:buildcache
+    environment:
+      # In development and testing, the SSO service needs to contact the Supabase
+      # service directly via Docker vs through the http://localhost/v1/supabase domain.
+      - SUPABASE_URL=http://kong:8000
```

Comment:
Let's use the staging database to get all the feeds. The URL is https://dev.supabase.telescope.cdot.systems/. I think it'll be fine because RLS is enabled so that it allows read right only.

Reply:
Check to make sure that you can override a single value...I don't think you can, it's all or none, since you can't remove keys this way.